Jenkinsfile-online

```diff
@@ -9,17 +9,16 @@ pipeline {
         string(name:'TAG_NAME',defaultValue: '',description:'')
     }
 
-    environment {
+   environment {
         DOCKER_CREDENTIAL_ID = 'dockerhub-id'
         GITHUB_CREDENTIAL_ID = 'github-id'
         KUBECONFIG_CREDENTIAL_ID = 'demo-kubeconfig'
         REGISTRY = 'docker.io'
-        DOCKERHUB_NAMESPACE = 'docker_username'
-        GITHUB_ACCOUNT = 'kubesphere'
+        DOCKERHUB_NAMESPACE = 'gongchao8888'
+        GITHUB_ACCOUNT = 'gongchao8888'
         APP_NAME = 'devops-java-sample'
-        SONAR_CREDENTIAL_ID = 'sonar-token'
+        SONAR_CREDENTIAL_ID= 'sonar-qube'
```

Comment:
In my view, `sonar-token` is pretty reasonable. Or you can consider changing it to `sonarqube-token`. And please don't remove the whitespace after `SONAR_CREDENTIAL_ID`